README.md

```diff
@@ -1 +1,97 @@
 # testcafe-angular-selectors
+
+This plugin provides selector extensions that make it easier to test Angular applications with [TestCafe](https://github.com/DevExpress/testcafe/). These extensions allow you to create a Selector to find elements on the page by **expressions** that are native to Angular applications (like `binding`, `model` etc.. ).
```

Comment:
@AlexanderMoskovkin wonders if it is correct to use the word `expression` here. From the Angular docs, I gathered that it's okay. @miherlosev , what do you think